pkgs/top-level/all-packages.nix

```diff
@@ -17225,8 +17225,8 @@ in
   bleachbit = callPackage ../applications/misc/bleachbit { };
 
   blender = callPackage  ../applications/misc/blender {
-    pythonPackages = python35Packages;
```

Comment:
Following https://wiki.blender.org/wiki/Building_Blender#Compiler_Versions?

Reply:
Doesn't seem like there is any particular reason to stick to gcc6 (did blender recommend it for 2.79?)

Reply:
Maybe add a comment pointing to that page for future reference.